catalog/gitops/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: gitops
+info:
+  description: This blueprint generates a GitOps CI/CD pipeline for use with ACM
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/apply-setters:v0.1
+      configMap:
+        cluster-name: cluster-name
```

Comment:
Are you planning to move these sections to configPath option ? Do you need fix command to provide an option, whether to create this in-line or in a separate fn-config file?

Reply:
Not currently planning to move them, unless you think that's a better default.

Reply:
The more I think about it, I feel that(using configPath) is the better default for multiple reasons
1. Kptfile doesn't support comments, so package publisher can't add descriptions as comments.
2. Merging pipeline section in Kptfile might lead to surprising results to users.
3. It will be easy to discover and pass the values with confidence if setter-values.yaml is a separate file, users need not open the Kptfile.

All these cases can be solved if we move them to setter-values.yaml and use configPath option.

cc @frankfarzan @mortent @droot

Reply:
If that should be the default, let's update the `fix` function to auto-generate a `setters.yaml` and reference that. I do agree it's easier to edit a separate file.